client/my-sites/no-results/index.jsx

```diff
@@ -13,7 +17,9 @@ class NoResults extends React.Component {
 	render() {
 		return (
 			<div className="no-results">
-				{ this.props.image ? <img className="no-results__img" src={ this.props.image } /> : null }
+				{ this.props.image ? (
+					<img className="no-results__img" src={ this.props.image } alt="" />
+				) : null }
```

Comment:
A more terse `{ image && <img> }` is good enough. React treats all falsy values equally.

We use the ternary with null a lot in Calypso 🙁 But it's just a stylistic nit.

Reply:
yeah, i originally redid this entire component, moving to a functional component with the useTranslate hook... But figured this was simpler to review.